db.js

```diff
@@ -291,7 +291,7 @@ export class ActivityPubDB extends EventTarget {
   async * iterateCollection (collectionOrUrl, { skip = 0, limit = DEFAULT_LIMIT, sort = 1 } = {}) {
     const collection = await this.#get(collectionOrUrl)
 
-    let items = collection.orderedItems || collection.items
+    let items = collection.orderedItems || collection.items || []
```

Comment:
![Screenshot 2024-05-03 at 7 32 59 PM](https://github.com/hyphacoop/reader.distributed.press/assets/68826419/897acae3-983f-4e77-aafd-12dc3c031634)

This was causing the follow button on Fediverse instances to freeze; I had to refresh the page to see the unfollow button.